ui/narrative/methods/export_json_to_staging/display.yaml

```diff
@@ -0,0 +1,50 @@
+name     : Export Object JSON To Staging Area
+subtitle :
+tooltip  : Export a KBase object's JSON represenation to the Staging Area.
+
+icon: data-yellow.png
+
+screenshots :
+    []
+
+suggestions :
+    apps:
+        related :
+            []
+        next :
+            []
+    methods:
+        related :
+            []
+        next :
+            []
+
+parameters :
+    input_ref :
+        ui-name: |
+            Data Object
+        short-hint : |
+            Select the input data object.
+
+    destination_dir:
+        ui-name: |
+            Destination Directory
+        short-hint : |
+            The destination directory to which the file will be saved.
+
+    format:
+        ui-name: |
+            Output format
+        short-hint : |
+            The format of the output zip file. standard is the output as returned from the workspace. legacy_data_import_export is the format that was returned from direct JSON downloads via the app panel prior to the coversion to this app.
```

Comment:
typo: "coversion" to this app -> "conversion" to this app

Reply:
Fixed